src/index.html

```diff
@@ -1,14 +1,51 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"
+          data-qa="hover">Laptops & computers</a>
```

Comment:
```suggestion
            <a 
              href="#" 
              class="nav_link"
              data-qa="hover"
            >
              Laptops & computers
            </a>
```

Reply:
still not fixed
<img width="635" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/4d40a252-9414-4a7b-91f2-2e9e46862f58">

Reply:
Strangely... because actually I've done it. I'll try to push it one more time